simplq/.eslintrc.json

```diff
@@ -19,6 +19,7 @@
     "react/destructuring-assignment": 0,
     "prefer-destructuring": 0,
     "import/no-named-as-default": 0,
-    "dot-notation": 0
+    "dot-notation": 0,
+    "react/jsx-props-no-spreading": 0
```

Comment:
Can we avoid this and explicitly pass down props?

I was [checking](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-props-no-spreading.md), and props spreading does affect readability.

If there is a good reason you need to do this in the PR, we can disable it inline where you are using it, but try to avoid.

Reply:
Yes, removed (react/jsx-props-no-spreading) and passed down props. Now it is readable.